### PR TITLE
Slight wording changes for Import-Styles.md

### PR DESCRIPTION
**Pull request by mtmail**
Given the huge PR and number of code and configuration changes, these wording suggestions are minuscule. Can't wait to test the new logic and adapt my local custom filters.

### CODE REVIEW

File: docs/customize/Import-Styles.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Configuring the Import of OSM data
+\# Configuring the Import of OSM data
```
Comment (lonvia) [inline docs/customize/Import-Styles.md:1]: What does that do?

File: docs/customize/Import-Styles.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ customize it.
 
 The core of Nominatim's flex import configuration is the `flex-base` module.
 It defines the table layout used by Nominatim and provides standard
-implementations for the import callbacks that help with customizing
+implementations for the import callbacks that help with customising
```
Comment (lonvia) [inline docs/customize/Import-Styles.md:26]: I was trying to settle on American spelling on the z vs. s front. In line with variable naming in the code.

File: docs/customize/Import-Styles.md (hunk 1 above)
Comment (mtmail) [reply docs/customize/Import-Styles.md]: It shows how easy it is to hit the \ key by accident. (removed now)

File: docs/customize/Import-Styles.md (hunk 2 above)
Comment (mtmail) [reply docs/customize/Import-Styles.md]: Ah, ok, I saw "recognise" and "initalise" later in the document and thought it was vice-versa.

**Pull request merged**